File: go-sdk/pkg/models/operations/deletecommentsid.go
package operations

import (
	"openapi/pkg/models/shared"
	"openapi/pkg/utils"
)

type DeleteCommentsIDPathParams struct {
	CommentID string `pathParam:"style=simple,explode=false,name=comment_id"`
}

type DeleteCommentsIDRequest struct {
	Retries    *utils.RetryConfig
	PathParams DeleteCommentsIDPathParams
}

type DeleteCommentsIDResponse struct {
	ClientError *shared.ClientError
	ContentType string
	StatusCode  int64
}

File: ts-sdk/src/sdk/models/operations/deletecommentsid.ts
import { SpeakeasyMetadata, SpeakeasyBase, RetryConfig } from "../../../internal/utils";
import * as shared from "../shared";



export class DeleteCommentsIdPathParams extends SpeakeasyBase {
  @SpeakeasyMetadata({ data: "pathParam, style=simple;explode=false;name=comment_id" })
  commentId: string;
}


export class DeleteCommentsIdRequest extends SpeakeasyBase {
  @SpeakeasyMetadata()
  retries?: RetryConfig;

  @SpeakeasyMetadata()
  pathParams: DeleteCommentsIdPathParams;
}


export class DeleteCommentsIdResponse extends SpeakeasyBase {
  @SpeakeasyMetadata()
  clientError?: shared.ClientError;

  @SpeakeasyMetadata()
  contentType: string;

  @SpeakeasyMetadata()
  statusCode: number;
}
